Sources/StreamChatUI/ChatMessageList/Attachments/ChatFileAttachmentListView+ItemView_Tests.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+@testable import StreamChatUI
+import UIKit
+import XCTest
+
+final class ChatFileAttachmentListViewItemView_Tests: XCTestCase {
+    private var fileAttachmentView: ChatMessageFileAttachmentListView.ItemView!
+    private var vc: UIViewController!
+    
+    override func setUp() {
+        super.setUp()
+        fileAttachmentView = ChatMessageFileAttachmentListView.ItemView().withoutAutoresizingMaskConstraints
+    }
+    
+    override func tearDown() {
+        fileAttachmentView = nil
+
+        super.tearDown()
+    }
+
+    func test_appearance_pdf() {
```

Comment:
We should ideally test different upload status state to see if the icon is set properly.